### PR TITLE
feature(about_page):RR-0017: about_page creation and routes

### PR DESCRIPTION
**Pull request by v1ljar**
Created the about page, added routes to About Us page. Added Poppins font to the project.

### CODE REVIEW

**Review by Jann337 — DISMISSED** [review]

File: radicals/urls.py [hunk 1]
```diff
@@ -21,13 +21,13 @@
...
     path('admin/', admin.site.urls),
-    path('', lambda request: HttpResponseRedirect('home/')),
-    path('backoffice/customers/', include('back_office.customers.urls')),
-    path('backoffice/invoices/', include('back_office.invoices.urls')),
-    path('backoffice/orders/', include('back_office.orders.urls')),
-    path('backoffice/products/', include('back_office.products.urls')),
-    path('backoffice/sales/', include('back_office.sales.urls')),
-    path('backoffice/settings/', include('back_office.settings.urls')),
+    path('', lambda request: HttpResponseRedirect('/home/')),
+    path('customers/', include('back_office.customers.urls')),
```
Comment (EimantasBlazevicius) [inline radicals/urls.py]: These should not be changed, change back to them having backoffice at the start as it was

**Review by EimantasBlazevicius — CHANGES_REQUESTED** [review]

File: radicals/urls.py (hunk 1 above)
Comment (v1ljar) [reply radicals/urls.py]: Fixed the urls.py and also found a issue with dropdown menu but it is solved now.

Comment (EimantasBlazevicius) [reply radicals/urls.py]: lines like this should not exist:
path('settings/', include('back_office.settings.urls')),
if it has backoffice in the include function, the backoffice should be present in the path string, like this:
path('backoffice/customers/', include('back_office.customers.urls')),

Delete the wrong ones

Comment (v1ljar) [reply radicals/urls.py]: Now should be ok.

**Review by EimantasBlazevicius — DISMISSED** [review]

**Review by T6nisValk — DISMISSED** [review]

**Review by Kapmann — DISMISSED** [review]

**Review by EimantasBlazevicius — APPROVED** [review]

**Review by T6nisValk — APPROVED** [review]

**Pull request merged**